.eslintrc

```diff
@@ -4,24 +4,72 @@
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
     "prettier",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript"
+  ],
+  "plugins": [
+    "prettier",
+    "@typescript-eslint",
+    "import"
   ],
-  "plugins": ["prettier", "@typescript-eslint", "simple-import-sort"],
```

Comment:
dev: drops `"simple-import-sort"`, which is largely covered by the `import` plugin.